rosidl_core_runtime/QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,116 @@
+This document is a declaration of software quality for the `rosidl_core_runtime` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
```

Comment:
should there be a quality declaration for `rosidl_code_generators` as well?
could there only be one in the top level?

Reply:
I'm not sure. There was _not_ a quality declaration in `rosidl_defaults`, so I did not create one here. It seems we only did quality declarations for the runtime packages.

Reply:
Okay, that makes sense to me